### DISCUSSION TITLE
Milestone on chart

### DISCUSSION THREAD

**Discussion by glatzl** [plausible/analytics#1712] [Anything Else]
Feature suggestion:
Be able to add a milestone to the chart. For example, new site design. 
![image](https://user-images.githubusercontent.com/29255598/155530585-b90ecc05-9d61-40aa-9777-0793f821d729.png)

**Comment by metmarkosaric** [comment]
thanks @glatzl! there's a same request already at https://github.com/plausible/analytics/discussions/97

**Reply by glatzl** [reply]
Alright! We can close this discussion then. Thanks for a great Google Analytics alternative.

**Reply by metmarkosaric** [reply]
thank you!
